lib/handler.js

```diff
@@ -210,20 +210,16 @@ Handler.prototype.logout = async function logout () {
   this.redirect(redirectUrl)
 }
 
-Handler.prototype.setTokens = async function setTokens () {
+Handler.prototype.setTokens = async function setTokens (accessToken, refreshToken) {
   await this.createSession()
 
-  const { accessToken, refreshToken, redirectUrl } = parse(this.req.url.split('?')[1])
-
   try {
     const token = await this.auth.createToken(accessToken, refreshToken, 'bearer')
 
     const refreshedToken = await token.refresh()
```

Comment:
Doesn't hurt but don't think this is necessary, if they expire the frontend will refresh them but given they're fresh from Identity it's unlikely they're close to expiry